egs2/mr_openslr64/asr1/setup.sh

```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/setup.sh
```

Comment:
This should not be included in specific recipe

Reply:
Resolved